tests/e2e/steps_submit_equivocation_proposal.go

```diff
@@ -21,7 +21,7 @@ func stepsRejectEquivocationProposal(consumerName string, propNumber uint) []Ste
 					ValPowers: &map[validatorID]uint{
 						validatorID("alice"): 509,
 						validatorID("bob"):   500,
-						validatorID("carol"): 495,
+						validatorID("carol"): 501,
```

Comment:
Why were those values changed?

Reply:
Some steps that changed the voting power were removed. Steps are executed sequentially, so if you remove one of them you have to account for the removal in other places.